src/components/datetime/DateTime.js

```diff
@@ -203,6 +203,9 @@ export default class DateTimeComponent extends Input {
     format += format.match(/z$/) ? '' : ' z';
     const timezone = this.timezone;
     if (value && !this.attached && timezone) {
+      if (Array.isArray(value) && this.component.multiple) {
+        return value.map((item) => _.trim(FormioUtils.momentDate(item, format, timezone).format(format))).join(', ');
+      }
       return _.trim(FormioUtils.momentDate(value, format, timezone).format(format));
     }
     return (value ? _.trim(moment(value).format(format)) : value) || '';
```

Comment:
What if timezone is undefined and component is multiple? The line 207 is skipped and we go to line 211 that does not expect array value.
It also would be nice to separate repeated code in some new method. I mean lines 207 and 209. The same is for Time component.

Reply:
I don't think that the repeated code should be separated in that case, because it is specific for that particular function.